check unsupported instructions when exporting mobile models

ann-ss opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#40738 check unsupported instructions when exporting mobile models**

Differential Revision: [D22298210](https://our.internmc.facebook.com/intern/diff/D22298210)

ann-ss committed:
check unsupported instructions when exporting mobile models

[ghstack-poisoned]

ann-ss committed:
Update on "check unsupported instructions when exporting mobile models"


Differential Revision: [D22298210](https://our.internmc.facebook.com/intern/diff/D22298210)

[ghstack-poisoned]

iseeyuan commented on test/mobile/test_lite_script_module.py:
```diff
@@ -66,6 +66,23 @@ def forward(self, arg):
+                return f.func(1, 2)
+
+        script_module = torch.jit.script(MyTestModule())
+        with self.assertRaisesRegex(RuntimeError, " is not supported in mobile module"):
```
Since we know which instruction is not supported, it's better to have the entire message comparison here.

ann-ss committed:
Update on "check unsupported instructions when exporting mobile models"


Differential Revision: [D22298210](https://our.internmc.facebook.com/intern/diff/D22298210)

[ghstack-poisoned]

ann-ss commented:
Moved to https://github.com/pytorch/pytorch/pull/40791 instead
